lib/src/components/AbstractDialog/AbstractDialog.tsx

```diff
@@ -148,6 +148,12 @@ const AbstractDialog = forwardRef<HTMLDialogElement, Props>((props, ref) => {
       // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
       const dialogEl = dialogRef.current!;
 
+      // Don't manage tab cycling when the focus is not within the dialog
+      // so that there is no key nav conflict when another modal is open.
+      if (!dialogEl.contains(document.activeElement)) {
```

Comment:
Does this negate the need for the `preventDefault` on line 168.  Or alternatively, would it have been better to adjust around that line rather than add this condition here?